rosco-manifests/src/test/java/com/netflix/spinnaker/rosco/manifests/helm/HelmTemplateUtilsTest.java

```diff
@@ -190,4 +204,177 @@ public void buildBakeRecipeSelectsHelmExecutableByVersion(
         Arguments.of("helm2", BakeManifestRequest.TemplateRenderer.HELM2),
         Arguments.of("helm3", BakeManifestRequest.TemplateRenderer.HELM3));
   }
+
+  @Test
+  public void buildBakeRecipeWithGitRepoArtifact(@TempDir Path tempDir) throws IOException {
+    // git/repo artifacts appear as a tarball, so create one that contains a helm chart.
+    addTestHelmChart(tempDir);
+
+    ArtifactDownloader artifactDownloader = mock(ArtifactDownloader.class);
+    RoscoHelmConfigurationProperties helmConfigurationProperties =
```

Comment:
Same comment as on the test that was converted from groovy...feels like it would be easier to just make a real `RoscoHelmConfigurationProperties` here and set the two fields rather than mock it.

Reply:
Done here: https://github.com/spinnaker/rosco/pull/740/commits/cf4b61deb559b3e41af2bcc0986ab75ba0a20487